pkg/Microsoft.ML/Microsoft.ML.nupkgproj

```diff
@@ -20,5 +20,7 @@
   <ItemGroup>
     <Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
     <Content Include="build\**\*" Pack="true" PackagePath="build" />
+    <Content Include="..\..\LICENSE" Pack="true" PackagePath="\" />
+    <Content Include="..\..\THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="\" />
```

Comment:
IMO this doesn't make sense because the `Microsoft.ML` nuget package doesn't contain `libmf` or any other 3rd party code.
Instead, the `Microsoft.ML.Microsoft.ML.MatrixFactorization` nuget contains this code, and it already has the `libmf` `COPYRIGHT` file inside of it - https://github.com/dotnet/machinelearning/blob/7fb76b026d0035d6da4d0b46bd3f2a6e3c0ce3f1/pkg/Microsoft.ML.MatrixFactorization/Microsoft.ML.MatrixFactorization.nupkgproj#L10

We use other 3rd party code in other places (TensorFLow, MKL) but `Microsoft.ML` doesn't contain that either. The individual packages that contain TensorFlow and MKL have the appropriate notices in them. The core package shouldn't need them.

Reply:
While technically true at this point in time, it doesn't seem to _hurt_ to include the third-party notices file now. Maintaining independent notices files for each package sounds problematic, and if a notice was added in the future for code impacting this package, it would not be expected for the file to be omitted.

Reply:
If this is the case, then why single out the `Microsoft.ML` nuget package?  Why not include the `THIRD-PARTY-NOTICES.TXT` in every nuget package built from this repo?

That appears to be the way `corefx` does it [here](https://github.com/dotnet/corefx/blob/04c675ebc3c78a175be1dfd5aa710c88cc389fc9/Packaging.props#L5) and [here](https://github.com/dotnet/corefx/blob/04c675ebc3c78a175be1dfd5aa710c88cc389fc9/Packaging.props#L51-L54). I just downloaded the `System.ComponentModel.Composition` nuget package and it has a 3rd party notice that includes the License notice for Brotli and the License notice for Unicode data.

Reply:
> If this is the case, then why single out the Microsoft.ML nuget package?

I didn't mean to single it out. I would suggest including it in all packages produced by this source repository.